pkg/products/threescale/prometheusRules.go

```diff
@@ -329,8 +333,78 @@ func (r *Reconciler) newAlertReconciler(logger l.Logger, installType string, ctx
 					},
 				},
 			},
+			{
+				AlertName: fmt.Sprintf("%s-missing-metrics", installationName),
```

Comment:
new alert here to cover metrics used in critical alerts that are created by the threescale reconciler. we have a similar pattern in cro as agreed. this should now cover the missing metrics at the right time.